example/w-flash/flash.eml.ml

```diff
@@ -31,8 +33,8 @@ let () =
     Dream.post "/"
       (fun request ->
         match%lwt Dream.form request with
-        | `Ok ["text", text] ->
-          let () = Dream.put_flash "Info" text request in
+          | `Ok ["text", text] ->
+            let () = Dream.put_flash "Info" text request in
```

Comment:
Why have these lines had an indentation-only change?

Also, where does this kind of indentation come from? It is wasteful of the left margin.